shotover-proxy/src/transforms/redis/cache.rs

```diff
@@ -33,10 +91,13 @@ pub struct TableCacheSchema {
 
 impl RedisConfig {
     pub async fn get_transform(&self, topics: &TopicHolder) -> Result<Transforms> {
+        let missed_requests = register_counter!("cache_miss");
```

Comment:
Remember to add docs for this new metric. See the other transform docs for an example.

Reply:
If we're planning to do it in a later PR I'd remove from this one.